core/http-auth-spi/src/main/java/software/amazon/awssdk/http/auth/spi/AuthOption.java

```diff
@@ -27,16 +27,16 @@
  * This is used in the output from the auth scheme resolver. The resolver returns a list of these, in the order the auth scheme
  * resolver wishes to use them.
  *
- * @see HttpAuthScheme
+ * @see AuthScheme
  */
 @SdkProtectedApi
-public interface HttpAuthOption {
+public interface AuthOption {
```

Comment:
sorry, I probably didn't call it out to you separately, but this was meant to be `AuthSchemeOption`